pkg/util/util.go

```diff
@@ -102,7 +102,7 @@ func AddDefaultRegistryToImage(image string) string {
 	}
 
 	for k := range commonDockerRegistries {
-		if strings.HasPrefix(image, k) || strings.HasPrefix(image, "gcr.io") || strings.HasPrefix(image, "k8s.gcr.io") {
+		if strings.HasPrefix(image, k) || strings.HasPrefix(image, "gcr.io") || strings.HasPrefix(image, "k8s.gcr.io") || strings.HasPrefix(image, "registry.k8s.io") {
```

Comment:
We don't this change as `registry.k8s.io` is already part of the common registries list. So the first condition `strings.HasPrefix(image, k)` should handle the condition that you are adding.